### PR TITLE
Troubleshooting doc change for setting the path variable on windows

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
Guides a user through verifying and setting their windows path variable to use the Salesforce CLI.

### What issues does this PR fix or reference?
@W-6330632@

### CODE REVIEW

File: docs/articles/troubleshooting.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ If you don’t see any SFDX commands in the command palette, make sure that you
 
 See: [Java Configuration](getting-started/java-setup)
 
+## Set Salesforce CLI Path (Windows)
+
+See [Installing the Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli). If after installing the cli you see the error 'Salesforce CLI is not installed', this likely means that the Salesforce CLI was not added to your path. Follow these steps for verifying and adding the Salesforce CLI to your Windows path variable.
+
+1. From your [Windows Search](../images/troubleshooting_set_cli_path.1.png) look for `Edit the system environment variables`.
```
Comment (lcampos) [inline docs/articles/troubleshooting.md]: I think the images should be added below the instructions. Right now they are links.

File: docs/articles/troubleshooting.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ If you don’t see any SFDX commands in the command palette, make sure that you
 
 See: [Java Configuration](getting-started/java-setup)
 
+## Set Salesforce CLI Path (Windows)
+
+See [Installing the Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli). If after installing the cli you see the error 'Salesforce CLI is not installed', this likely means that the Salesforce CLI was not added to your path. Follow these steps for verifying and adding the Salesforce CLI to your Windows path variable.
```
Comment (lcampos) [inline docs/articles/troubleshooting.md]: Replace `cli` with CLI or Salesforce CLI

File: docs/articles/troubleshooting.md (hunk 1 above)
Comment (sfsholden) [reply docs/articles/troubleshooting.md]: Okay, I will adjust. I wasn't sure if that would be best since none of the other sections have images.

**Comment by sfsholden** [comment]
Reopening for suggested changes from Bindu.

**Pull request merged**